app/assets/stylesheets/listings/_index.scss

```diff
@@ -0,0 +1,4 @@
+.container-display {
+  margin: 60px 10px;
+  margin-bottom: 80px;
+}
```

Comment:
If the margin property has three values:

`margin: 25px 50px 75px;`
top margin is 25px
right and left margins are 50px
bottom margin is 75px